app/components/RemixLink.tsx

```diff
@@ -0,0 +1,60 @@
+import { useHref } from "@remix-run/react";
+import type { LinkProps } from "@remix-run/react";
+import React, { forwardRef } from "react";
+import type { Ref } from "react";
+import { useLinkClickHandler } from "react-router-dom";
+import { Button, Link } from "@navikt/ds-react";
+
+interface IProps extends LinkProps {
+  as: "Link" | "Button";
+  children: React.ReactElement | string;
```

Comment:
Istedet for å definere children i props kan du bruke i komponenten `PropsWithChildren<IProps>` :)

Reply:
Jepp, enig! Grunnen til at det ikke ble sånn i første omgang er at jeg aldri husker syntaksen, og dermed gjorde jeg det på denne måten. Fikset! 😄